deps/rabbit/src/rabbit_tracking.erl

```diff
@@ -37,10 +38,29 @@
 
 id(Node, Name) -> {Node, Name}.
 
--spec count_tracked_items(function(), integer(), term(), string()) ->
+-spec count_tracked_items_ets(atom(), term(), string()) ->
     non_neg_integer().
+count_tracked_items_ets(Tab, Key, ContextMsg) ->
+    lists:foldl(fun (Node, Acc) when Node == node() ->
+                        N = case ets:lookup(Tab, Key) of
+                                []         -> 0;
+                                [{_, Val}] -> Val
+                            end,
```

Comment:
Could you please do the same as you did for `list_on_node()` and use a function to count the local table, then use this function in the RPC call below?